spec/factories/news.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+
+  factory :news do
+    title 'title'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.